src/sidebar/containers/container.js

```diff
@@ -48,12 +48,20 @@ class AbstractTabContainer {
         this.element.addEventListener('dragleave', (e) => {
             if (!e.currentTarget || !e.currentTarget.classList) return
             e.currentTarget.classList.remove('container-dragged-over')
-        })
+            e.currentTarget.classList.remove('tab-dragged-over')
```

Comment:
Could you change the indention to spaces? The codebase uses space indention, and while I prefer tabs I would rather not mix the two.

Reply:
Sure.